graph/telemetry/istio/appender/mesh_check.go

```diff
@@ -98,3 +109,15 @@ func (a *SidecarsCheckAppender) namespaceOK(namespace string, namespaceInfo *gra
 	}
 	return false
 }
+
+// isAmbientNs returns true if the namespace has the ambient annotations
+func (a *MeshCheckAppender) isAmbientNs(namespace string, cluster string, globalInfo *graph.AppenderGlobalInfo) bool {
+	ns, _ := globalInfo.Business.Namespace.GetClusterNamespace(context.TODO(), namespace, cluster)
+
+	for _, label := range ns.Labels {
+		if label == "istio.io/dataplane-mode" {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
This is fine but I think it could be better if `models.Namespace` included an `IsAmbient` flag, which was set one time, in a centralized location.  Then the consumers of `models.Namespace` don't need to check repeatedly, and don't need to know how to check if a namespace isAmbient.  What do you think?

Reply:
That's a good idea. I have added the logic to models.Namespace